shell/platform/android/android_surface_gl.cc

```diff
@@ -149,4 +149,6 @@ intptr_t AndroidSurfaceGL::GLContextFBO() const {
   return 0;
 }
 
+void AndroidSurfaceGL::SetFlutterView(const fml::jni::JavaObjectWeakGlobalRef& flutter_view) {}
```

Comment:
Can you make sure you run clang-format over all the modified files please? You can configure your editor to automatically format CC files on save.